src/plugins/intel_cpu/src/utils/print_model.hpp

```diff
@@ -0,0 +1,424 @@
+        std::replace(name.begin(), name.end(), '-', 'n');
+        if (name[0] >= '0' && name[0] <= '9') {
+            const auto& type_info = op->get_type_info();
+            name.insert(0, type_info.name);
+        }
+        int idx = 0;
+        if (opname_count.count(name)) {
+            idx = opname_count[name] + 1;
+        }
+        opname_count[name] = idx;
+
+        if (idx)
+            name += std::to_string(idx);
+
+        opname[op.get()] = name;
+    }
+
+    for (auto op : f.get_ordered_ops()) {
+        if (literal_consts.count(op))
+            continue;
+
+        const auto& type_info = op->get_type_info();
+        auto version_info = std::string(type_info.get_version());
+        auto type = version_info + "::" + type_info.name;
+        auto name = opname[op.get()];
+        os << prefix << "    ";
+
+        if (auto constop = std::dynamic_pointer_cast<op::v0::Constant>(op)) {
+            os << "auto " << name << " = makeConst(" << to_code(op->get_output_element_type(0)) << ", "
+               << to_code(op->get_output_shape(0)) << ", " << to_code(constop, true) << ");" << std::endl;
+        } else {
+            os << "auto " << name << " = makeOP<" << type << ">({";
+            // input args
+            sep = "";
+            for (size_t i = 0; i < op->get_input_size(); i++) {
+                auto vout = op->get_input_source_output(i);
+                auto iop = vout.get_node_shared_ptr();
+                if (iop->get_output_size() > 1) {
+                    auto out_port = vout.get_index();
+                    os << sep << tag << opname[iop.get()] << "->output(" << out_port << ")";
+                } else {
+                    if (literal_consts.count(iop))
+                        os << sep << tag << literal_consts[iop];
+                    else
+                        os << sep << tag << opname[iop.get()];
+                }
+                sep = ", ";
+            }
+            os << "}";
+
+            // attributes as AnyMap
+            std::stringstream ss2;
+            OstreamAttributeVisitor osvis(ss2);
+            op->visit_attributes(osvis);
+            auto str_attr = ss2.str();
+            if (str_attr.size())
+                os << ", {" << str_attr << "}";
+            os << ");   //  tensor_array<" << get_output_values_info(op) << "> " << op->get_friendly_name();
+
+            os << "(";
+            sep = "";
+            for (size_t i = 0; i < op->get_input_size(); i++) {
+                auto vout = op->get_input_source_output(i);
+                auto iop = vout.get_node_shared_ptr();
+                os << sep << tag << iop->get_friendly_name();
+                if (iop->get_output_size() > 1) {
+                    auto out_port = vout.get_index();
+                    os << "[" << out_port << "]";
+                }
+                sep = ", ";
+            }
+            os << ")" << std::endl;
+
+            //==========================================================
```

Comment:
Please cleam up commented code in this file

Reply:
OK